lr-cli/cortx_setup/commands/node/prepare/server.py

```diff
@@ -0,0 +1,68 @@
+#
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+#
+
+import os
+from provisioner.salt import (
+    cmd_run,
+    local_minion_id,
+    StateFunExecuter,
+    StatesApplier
+)
+from provisioner.config import PRVSNR_ROOT_DIR
+from .config import NodePrepareServerConfig
+from ...common_utils import get_pillar_data
+
+
+class NodePrepareServer(NodePrepareServerConfig):
+
+    def _ip_not_pingable(self, ip: str):
+        res = os.system(f"ping -c 1 {ip}")
```

Comment:
can we use cmd_run here ?

Reply:
here it is expected that ip should not be pingable.
so if using cmd_run in case of non-zero retcode it fails directly.
i tried modifying the command but it didn't help.